engine/websocketroutine_manager.go

```diff
@@ -287,6 +284,9 @@ func (m *WebsocketRoutineManager) websocketDataHandler(exchName string, data int
 		}
 		m.syncer.PrintOrderbookSummary(base, "websocket", nil)
 	case *order.Detail:
+		if !m.orderManager.IsRunning() {
```

Comment:
I think you should add this to `case []order.Detail` below

Reply:
Agreed and fixed!